biosys/apps/main/api/urls.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import absolute_import, unicode_literals, print_function, division
+
+from django.conf.urls import url
+from rest_framework import routers
+
+from main.api import views as main_views
+
+
+router = routers.DefaultRouter()
+router.register(r'projects?', main_views.ProjectViewSet, 'project')
+router.register(r'sites?', main_views.SiteViewSet, 'site')
+router.register(r'datasets?', main_views.DatasetViewSet, 'dataset')
+router.register(r'genericRecords?', main_views.GenericRecordViewSet, 'genericRecord')
```

Comment:
I don't know. I changed my mind a couple of times. I wanted to match the model name, but now I realize that snake case is probably better for URLs. I will change that.